po/zh-TW.po

```diff
@@ -1,1152 +1,1172 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Comprehensive Rust 🦀\n"
-"POT-Creation-Date: 2023-08-23\n"
-"PO-Revision-Date: 2023-06-02 14:43+0800\n"
-"Last-Translator: Hank Chen <kuanhungchen@google.com>\n"
+"POT-Creation-Date: \n"
```

Comment:
@kochinc, please make sure to run `cargo install mdbook-i18n-helpers`. That will give you version 0.2.4, which in turns adds this importand header.

Reply:
See #1243 for what it does: in basically ensures that you are translating against a fixed version of the course.